tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -591,58 +586,89 @@ constexpr bool test() {
+        static_assert(is_constructible_v<span<const int, 3>, const ContiguousSizedRange&>);
         static_assert(!is_constructible_v<span<double>, ContiguousSizedRange&>);
+        static_assert(!is_constructible_v<span<double, 3>, ContiguousSizedRange&>);
 
         static_assert(is_convertible_v<ContiguousSizedRange&, span<int>>);
+        static_assert(is_convertible_v<ContiguousSizedRange&, span<int, 3>>);
         static_assert(is_convertible_v<ContiguousSizedRange&, span<const int>>);
+        static_assert(is_convertible_v<ContiguousSizedRange&, span<const int, 3>>);
         static_assert(is_convertible_v<const ContiguousSizedRange&, span<const int>>);
+        static_assert(is_convertible_v<const ContiguousSizedRange&, span<const int, 3>>);
 
         static_assert(is_constructible_v<span<Base>, BasicRange<Base>&>);
+        static_assert(is_constructible_v<span<Base, 3>, BasicRange<Base>&>);
         static_assert(!is_constructible_v<span<Base>, BasicRange<Derived>&>);
+        static_assert(!is_constructible_v<span<Base, 3>, BasicRange<Derived>&>);
 
         ContiguousSizedRange user_range;
 
         span<int> sp_1(user_range);
         span<const int> sp_2(user_range);
         span<const int> sp_3(as_const(user_range));
 
+        span<int, 3> sp_4(user_range);
+        span<const int, 3> sp_5(user_range);
+        span<const int, 3> sp_6(as_const(user_range));
+
         assert(sp_1.data() == user_range.data());
         assert(sp_2.data() == user_range.data());
         assert(sp_3.data() == user_range.data());
+        assert(sp_4.data() == user_range.data());
+        assert(sp_5.data() == user_range.data());
+        assert(sp_6.data() == user_range.data());
 
         assert(sp_1.size() == 3);
         assert(sp_2.size() == 3);
         assert(sp_3.size() == 3);
+        assert(sp_4.size() == 3);
+        assert(sp_5.size() == 3);
+        assert(sp_6.size() == 3);
 
         FunctionTakingSpan<int>(user_range);
         FunctionTakingSpan<const int>(user_range);
         FunctionTakingSpan<const int>(as_const(user_range));
 
 #ifdef __cpp_lib_concepts
         static_assert(!is_constructible_v<span<int>, ContiguousSizedRange>);
+        static_assert(!is_constructible_v<span<int, 3>, ContiguousSizedRange>);
         static_assert(is_constructible_v<span<int>, SafeContiguousSizedRange>);
+        static_assert(is_constructible_v<span<int, 3>, SafeContiguousSizedRange>);
         static_assert(is_constructible_v<span<const int>, ContiguousSizedRange>);
+        static_assert(is_constructible_v<span<const int, 3>, ContiguousSizedRange>);
 
         static_assert(is_convertible_v<SafeContiguousSizedRange, span<int>>);
+        static_assert(is_convertible_v<SafeContiguousSizedRange, span<int, 3>>);
         static_assert(is_convertible_v<ContiguousSizedRange, span<const int>>);
+        static_assert(is_convertible_v<ContiguousSizedRange, span<const int, 3>>);

```

Comment:
Ditto, these tests look like they shouldn't be passing due to conditional explicit.

Reply:
Yep